server.go

```diff
@@ -95,6 +95,10 @@ const (
 	// value used or a particular peer will be chosen between 0s and this
 	// value.
 	maxInitReconnectDelay = 30
+
+	// multiAddrConnectionStagger is the number of seconds to wait between
+	// attempting to a peer with each of its advertised addresses.
+	multiAddrConnectionStagger = 10
```

Comment:
Should have this me a `time.Duration` otherwise I don't think it'll actually delay much, since 10 will end up being converted to nanoseconds

Reply:
indeed 🙈